### ISSUE TITLE
Suggestion: aria-live triggers (certain attributes)

### ISSUE THREAD

**Issue by aMadReason** [w3c/aria-practices#2668]
The Carousel pattern suggests using aria-live and atomic, I've found unless I dynamically add and remove 'display: none' aria-live doesn't read content updates (this is unaffected by what i set in aria-relevant) -> this can be very disruptive when using animations or scrolling. 

It might be beneficial to trigger aria-live announcements when certain attributes change, e.g.  aria-hidden, inert

Sceneraio: Carousel slides all loaded for grid layout, inactive slides have 'inert' attribute and the parent component that acts as scroller has aria-live and aria-atomic. When a new slide is made active there are subtle animations. Having to apply toggle display: none on elements will cause layout issues and require the removal of animations.

Please see the examples here that by a google dev that follow the pattern, but as it doesn't use display: none the aria-live doesn't announce changes: https://gui-challenges.web.app/carousel/dist/

Video: https://www.youtube.com/watch?v=CXJv6zM003M

(I didn't do the video or examples but found the sourcecode for the examples to be a good example of this)

**Comment by mcking65** [comment]
@aMadReason 

We would definitely like to address this problem!! This issue is essentially the same as #78, the issue referenced above by @JAWS-test.

Unfortunately, there is only one way to make live announcements that work with JAWS, NVDA, VoiceOver, and Narrator, and it is not an approach that is appropriate for carousel. That way is to use role alert.

The sad reality is that live regions are important but do not yet have robust support at any level of the AT rendering stack. That is why issue #78 remains unresolved. This situation has the attention of the ARIA Working Group and is one of the major topics in our upcoming face-to-face workshop in May.

At this time, treating live content as a nice-to-have feature is best. That is certainly the case for the carousel pattern. It is accessible without the live content. The live content simply makes quickly skimming the carousel easier.

Closing this as a duplicate of #78.
